data/_models/shared/Catalogues.csv

```diff
@@ -1,7 +1,7 @@
 tableName,tableExtends,columnName,columnType,key,required,refSchema,refTable,refLink,refBack,validation,semantics,description,profiles,visible,computed
 Catalogues,,,,,,,,,,,dcat:Catalog,A collection of resources within a network or consortium or about a common topic,DataCatalogueFlat,,
-Catalogues,,name,,1,true,,,,,,dct:title,The name given to the catalogue,DataCatalogueFlat,,
-Catalogues,,network,ref,1,true,,Resources,,,,,Network or consortium that publishes this catalogue,DataCatalogueFlat,,
-Catalogues,,type,ontology,,true,CatalogueOntologies,CatalogueTypes,,,,dcat:theme,Type of catalogue,DataCatalogueFlat,,
```

Comment:
waarom theme eraf gehaald?

Reply:
of vind je dit geen 'theme'. Let wel, RDF heeft er geen enkel probleem mee als wij meerdere velden naar hetzelfde dcat item mappen. Die theme tags komen voor de buitenwereld gewoon in 1 lijstje.

Reply:
dcat:Catalog class does not have a dcat:theme, but dcat:themeTaxonomy. And this is a controlled vocabulary. So we should just write it in the rdf, I guess. https://healthdcat-ap.github.io/#Catalogue.themes says: "This property refers to a knowledge organisation system used to classify the Catalogue's Datasets. It must have at least the value NAL:data-theme as this is the mandatory controlled vocabulary for dcat:theme."

Reply:
And you are right we could have this 'NAL:data-theme' hardcoded?